build.gradle

```diff
@@ -159,7 +159,7 @@ def toPackageClasspath = files(
 sourceSets {
     main {
         java {
-            srcDir "src"
+            srcDirs = ["src", "tests"]
```

Comment:
I don't think marking tests directory as source sets is good because it will also bring tests(eg. OsTrustedTest, InterningTest etc.) to released jar, which is not interesting to the person who uses the inference jar. I recommend using checker framework's way of handling this problem, as in the [link](https://github.com/opprop/checker-framework/blob/master/framework/src/org/checkerframework/framework/test/CheckerFrameworkPerFileTest.java). The basic idea is to only move necessary classes into source sets and test files that are not part of testing framework still remain in tests directory.